config/meltor.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+use Illuminate\Support\Facades\DB;
+
+return [
+
+    // Names of the Laravel connections, used as the base for the new, generated migration.
+    'connection' => [
+
+        // Connection to read the tables.
+        'data'   => fn() => DB::getDefaultConnection(),
```

Comment:
A closure instead of the standard Laravel way of providing a connection name? 
Why not default to null, and have the code use the defaultConnection when null instead?

Reply:
sure the code could do that, but this is at the same time a template for those who want to customize this. it shows in a simple manner how one could provide a different connection.

that being said i need to figure out if anyone would need a closure, or if a string should do it.